common/src/main/java/uk/nhs/adaptors/common/util/CodeableConceptUtils.java

```diff
@@ -13,4 +14,9 @@ public static CodeableConcept createCodeableConcept(String code, String system,
             .setDisplay(display);
         return codeableConcept;
     }
+
+    public static CodeableConcept getCodeableConceptWithCoding(String system, String code, String display) {
```

Comment:
Sorry Rory, I was thinking that you could use already existing method `createCodeableConcept`. It already creates `coding`, but a bit differently by doing `getCodingFirstRep()` and then setting the code and the system. It also sets text, but you can just pass null there.